syntax = "proto3";

package syft.grid.messages;

import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";


message SuccessResponseMessage {
  syft.core.common.UID msg_id = 1;
  string resp_msg = 3;
  syft.core.io.Address address = 4;
}
